spec/controllers/spree/admin/orders/payments/payments_controller_spec.rb

```diff
@@ -40,7 +40,7 @@
           spree_post :create, payment: params, order_id: order.number
 
           redirects_to_list_of_payments_with_success_flash
-          expect(order.reload.payments.last.state).to eq "checkout"
+          expect(order.reload.payments.last.state).to eq "pending"
```

Comment:
This is a meaningful change

Reply:
This may be confusing for users that use it a lot because now order payments will display this state once placed. Something to discuss with product/Filipe.

Reply:
:warning: this could clash with https://github.com/openfoodfoundation/openfoodnetwork/blob/7b87476986aeb0db64624bdb661fcf4e63e37357/app/services/order_syncer.rb#L58-L69. It checks for checkout state.

Reply:
let's see if what I found out makes sense. This code above checks for unprocessed payments placed with the subs' payment method that was just changed (these are the sort of things Order Syncer syncs). Since the scope of the present PR is backoffice payments only, there are no consequences that could affect subscriptions.

While investigating I found out we check for `checkout` because that's the initial state of the payment's state machine. Yes, this code could do better at being a bit more intention revealing and removing duplication. It's the [SubscriptionConfirmJob#process_payment!](https://github.com/openfoodfoundation/openfoodnetwork/blob/b1324c62698867f9c1870b302ec579be146a72c1/app/jobs/subscription_confirm_job.rb#L52) which moves the payment through the state machine.

So, tl;dr. I think we're safe here @andrewpbrett